ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -48,7 +49,8 @@ type ssh_daemon_options_type = {
     "ClientAliveInterval"               ? long
     "DenyGroups"                        ? string
     "DenyUsers"                         ? string
-    "HostKey"                           ? string
+    "GSSAPIStrictAcceptorCheck"         ? ssh_yesnostring
+    "HostKey"                           ? string[]
```

Comment:
I've changed multiline options to have its own type, and options/HostKey back to string. So hopefully this is completely backwards compatible.

I could do an sshd_hostkeys type, but I guess it would require something specifically for handling HostKey. Would it go under component/ssh/mutliline_options or component/ssh? We could do something slightly more generic when it encounters a hash, but I'm not sure discarding the keys (rsa, dsa etc) is particularly generic. I think I would prefer keeping things simple and consistent (like they are now), but I'm happy to do it this way too.